protocol/messenger_mailserver.go

```diff
@@ -294,12 +294,12 @@ func (m *Messenger) resetFiltersPriority(filters []*transport.Filter) {
 	}
 }
 
-func (m *Messenger) RequestAllHistoricMessagesWithRetries() (*MessengerResponse, error) {
-	return m.performMailserverRequest(m.RequestAllHistoricMessages)
+func (m *Messenger) RequestAllHistoricMessagesWithRetries(forceFetchingBackup ...bool) (*MessengerResponse, error) {
+	return m.performMailserverRequest(func() (*MessengerResponse, error) { return m.RequestAllHistoricMessages(forceFetchingBackup...) })
 }
 
 // RequestAllHistoricMessages requests all the historic messages for any topic
-func (m *Messenger) RequestAllHistoricMessages() (*MessengerResponse, error) {
+func (m *Messenger) RequestAllHistoricMessages(forceFetchingBackup ...bool) (*MessengerResponse, error) {
```

Comment:
can we replace `forceFetchingBackup ...bool` with `forceFetchingBackup bool` ?

Reply:
Well, ofc we can do whatever we want, but this way I made it optional, means if it's provided then provided value will be used, otherwise it defaults to `false`.

Reply:
why we need optional

Reply:
@qfrank well, firstly the default behavior of this function remains as it was and that is an intended behaviour, setting a `forceFetchingBackup` flag is very specific case, hence not sure why should we bother one who is going to use this function with setting a `forceFetchingBackup` flag, but those who really need it for their work they will know about it and will set it if they really want.   

Why do you think that it's so crucial to make it non optional flag?